app/models/form/sales/pages/person.rb

```diff
@@ -0,0 +1,14 @@
+class Form::Sales::Pages::Person < ::Form::Page
+  def initialize(id, hsh, subsection, person_index)
```

Comment:
As `person_index` is just a magic number, please could you make it a keyword argument?